raspiaudio-muse-luxe.yaml

```diff
@@ -171,15 +269,21 @@ binary_sensor:
           - voice_assistant.stop:
 
 light:
-  - platform: fastled_clockless
+  - platform: neopixelbus
```

Comment:
Try `esp32_rmt_led_strip`, see the [muse proto](https://github.com/esphome/media-players/blob/main/raspiaudio-muse-proto.yaml#L130) for config.

Reply:
I did, but are there any benefits I could read up on?